src/main/java/entity/User.java

```diff
@@ -0,0 +1,37 @@
+package entity;
+
+import javax.swing.text.Position;
+
+public class User {
+
+    private final String username;
+    private final String password;
+    private final double balance;
+    private Portfolio portfolio;
+    private TransactionHistory transactionHistory;
+
+    // TODO: difficulty level?
+    // TODO: password need to be encrypted?
+
+    public User(String username, String password, double balance) {
```

Comment:
We may also need to pass a `Portfolio` and `TransactionHistory` into user constructor. Even though this creates a strong dependency between these classes, I think it still logically makes sense, as `Portfolio` and `TransactionHistory` never exist independently from `User`